modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/CPInstancePriceEntryClayTableDataSetDisplayView.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.pricing.web.internal.frontend;
```

Comment:
@riccardo-alberti based off https://github.com/brianchandotcom/liferay-portal/pull/98680#issuecomment-777707272, looks like we should have the package path matching the same as `extends BaseTableClayDataSetDisplayView`.
```suggestion
package com.liferay.commerce.pricing.web.internal.frontend.taglib.clay.data.set.view.table;
```